sktime/forecasting/naive_variance.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
```

Comment:
can you pls add this to `naive.py` instead of having a new file

Reply:
it's a separate estimator because it should be able to work with all other forecasters (allowing us to add naive variance strategy to any forecaster).

Reply:
ah right I see what you mean, I will move it there.